content/tutorials/manage-the-rippled-server/installation/build-run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,362 @@
+2021-Dec-09 21:31:53.291747 UTC LoadManager:DBG Starting
+2021-Dec-09 21:31:53.291846 UTC gRPC Server:NFO Starting gRPC server at 0.0.0.0:60051
+2021-Dec-09 21:31:53.293246 UTC LedgerCleaner:DBG Started
+2021-Dec-09 21:31:53.295543 UTC ReportingETL::ETLSource:DBG handleMessage : Received a message on ledger  subscription stream. Message : {
+   "result" : {},
+   "status" : "success",
+   "type" : "response"
+}
+ - { validated_ledger :  , ip : 127.0.0.1 , web socket port : 6006, grpc port : 50051 }
+2021-Dec-09 21:31:53.368075 UTC ReportingETL:NFO monitor : Database is empty. Will download a ledger from the network.
+2021-Dec-09 21:31:53.368183 UTC ReportingETL:NFO monitor : Waiting for next ledger to be validated by network...
+```
+
+## Frequently Asked Questions
+
+**Do I need to run more than one instance of `rippled` in order to use Reporting mode?**
+
+Yes. A `rippled` server running in reporting mode does not connect to the P2P network, but instead extracts validated data from one or more `rippled` servers that are connected to the P2P network. In addition to the `rippled` server running in reporting mode, you need to run at least one other `rippled` server that is connected to the P2P network. 
+
+**I’ve already installed `rippled`. Can I update the configuration file to enable reporting mode and restart `rippled`?** 
+
+No. Currently, you need to download the source and build rippled for reporting mode. 
+There are initiatives in progress to provide packages for reporting mode.
+
+
+**To run `rippled` in Reporting mode, I need at least one `rippled` server running in P2P mode in addition to `rippled` in Reporting mode. Does this mean I need double the disk space?** 
+
+The answer depends on the location of your primary datastore. If you use Cassandra as the primary datastore, the disk space requirement to run `rippled` in reporting mode will be lower as the data is not stored on your local disk. The PostgreSQL server can be remote as well.
+
+Lastly, the P2P mode server only needs to keep very recent history, while the reporting mode server keeps long term history.
+
+For more information on system requirements to run `rippled`, see https://xrpl.org/system-requirements.html. 
+
+**How can I confirm the validity of the data that comes from the Postgres or Cassandra database?**
+
+`Rippled` running in Reporting mode only serves validated data from the ETL source specified in the config file. If you are accessing someone else's `rippled` server in P2P mode as the ETL source, you are implicitly trusting that server. If not, you need to run your own `rippled` node in P2P mode.
+
+**Is it possible to make traditional SQL queries to the relational database rather than using the API?**
+
+Technically, you *can* directly access the database if you want. However, the data is hashed as blobs and you have to decode the blobs in order to access using traditional SQL queries.
```

Comment:
```suggestion
Technically, you *can* directly access the database if you want. However, the data is stored as binary blobs and you have to decode the blobs to access the component fields. This makes traditional SQL queries much less useful since they cannot find and filter the individual fields of the data.
```